7_react/react1/week2/todo_list_app/src/components/form.jsx

```diff
@@ -0,0 +1,81 @@
+import { v4 as uuidv4 } from "uuid";
+import { useEffect } from "react";
+
+const Form = ({
+  input,
+  setInput,
+  todos,
+  setTodos,
+  deadline,
+  setDeadline,
+  editTodo,
+  setEditTodo,
+}) => {
+  const updateTodo = (description, id, completed, deadline) => {
```

Comment:
The update logic is broken right now, please look into it

https://github.com/HackYourFuture-CPH/class27-homework/assets/48878547/e1b9afcf-dbe3-48a4-8ebe-23a329be8393